src/package/store.rs

```diff
@@ -139,6 +139,24 @@ impl PackageStore {
         Ok(manifest)
     }
 
+    /// Validate this package
+    #[cfg(feature = "validation")]
+    pub async fn validate(
```

Comment:
Can we move this to `Package`?

Reply:
`Package` describes a package that is in-memory. Unfortunately we can only do the parsing by reading from the filesystem. So even though `PackageStore` is kind of the wrong place, it is the place where we interact even with the local package on disk.